README.md

```diff
@@ -1,41 +1,149 @@
-# Servicer
-CI/CD Automation Framework
+# Servicer #
 
-The primary goal of this framework is to completely automate creating, updating, and removing an environment (a collection of services). To accomplish this, this framework executes in your CI environment (eg: Bitbucket Pipelines, Gitlab, Jenkins, CircleCI, etc) and interfaces directly with cloud infrastructure provider libraries (AWS, Google Cloud, Azure, etc).
+Servicer is a CI framework that runs inside popular CI platforms, such as Bitbucket Pipelines, Jenkins, and CircleCI. It also allows you to run and debug your CI jobs locally.
+
+Servicer works with the *services* you define in `.servicer/services.yaml`, and takes *steps* to complete *jobs*.
 
 ## Separation of Concerns ##
-Servicer is separated into 3 main components:
+Servicer is separated into three main components: adapters, services, and the project environment.
+
+*Adapters* allow Servicer to use or interact with external systems, like Google Cloud Platform, GitHub, and Jenkins. Three kinds of adapters exist: CI adapters, auth adapters, and service adapters. These are written as reusable python modules, and are located in `servicer/builtin`. 
 
-### Adapters ###
-CI Adapters, Auth Adapters, and Service Adapters are written as re-usable python modules. These modules do not carry any project specific concepts, and are designed to be an extendable, re-usable framework for interfacing with any cloud service provider.
+ *Services* are the building blocks of your project. Services are defined in the .servicer/services.yaml file in your project, and these definitions include the configuration and commands needed to build, test and deploy each service. Additionally, service dependencies may be defined to automate the order services and steps are executed in.
 
-### Project Service Configuration ###
-Your project's `services.yaml` file represents the settings and service relationships unique to your project. This configuration resides in source control.
+The *project environment* should contain environment variables for holding credentials and other secrets that should not be committed to source control. These can be configured on your CI tool, or, if you are running Servicer locally, from a `.servicer/*.env.yaml` file.
 
-### Environment Settings ###
-Any environment specific credentials or secrets should live in simple environment variables, which can be read and inserted into the build at build-time. These secrets should never be checked into source control, and instead be populated in the build environment.
+## Using Servicer ##
+For the simplest use case, use the command below. With it, Servicer's *job* will be to take every step for every service you've defined in `.servicer/services.yaml`.
+```
+servicer
+```
+If you want Servicer to execute all steps for one or more specific services, like my-docker-image and my-python-package, specify those services in the command.
```

Comment:
I messed up the hyphens/underscores in my previous comment, these should be changed to match the examples below: `my_docker_image` and `my_python_package`.